kernel/src/main/scala/org/apache/toree/boot/CommandLineOptions.scala

```diff
@@ -100,6 +98,13 @@ class CommandLineOptions(args: Seq[String]) {
     "interpreter-plugin"
   ).withRequiredArg().ofType(classOf[String])
 
+  private val _spark_context_intialization_timeout = parser.accepts(
+    "spark-context-intialization-timeout",
+    "The time (in milliseconds) allowed for creation of the spark context. " +
+      "Failure to create a context in this time could result in duplicate initialization messages. " +
+      "The default value is 100 milliseconds."
+  ).withRequiredArg().ofType(classOf[Int]).defaultsTo(100)
```

Comment:
This should default to a configuration value, not a hard-coded 100ms. We can set the 100ms default in Toree's default config. That way this behaves like other options.

Reply:
I see the pattern in `resources/compile/application.conf` and `resources/compile/reference.conf` for other options with default values. I will just follow the pattern for the new option.